components/nds-filters/src/FilterSummary.js

```diff
@@ -0,0 +1,165 @@
+import React from "react";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+import { Tag } from "@nice-digital/nds-tag";
+import RemoveIcon from "@nice-digital/icons/lib/Remove";
+import "./../scss/filter-summary.scss";
+
+export function FilterSummary({
+	children,
+	className,
+	sorting = [],
+	activeFilters = [],
+	headingLevel = 2,
+	...rest
+}) {
+	const HeadingLevel = "h" + headingLevel;
+
+	return (
+		<div className={classnames("filter-summary", className)} {...rest}>
+			<div className="filter-summary__count">
+				<HeadingLevel className="h5 mv--0">{children}</HeadingLevel>
+			</div>
+			{sorting.length && (
```

Comment:
@johndavey72 - another for your PR today pls: this is rendering a `0` if there are no items:
![image](https://user-images.githubusercontent.com/893676/134164504-2d36a23d-9700-4ffa-a317-56b57165d54d.png)

Reply:
leave it with me